scipy/signal/spectral.py

```diff
@@ -26,30 +27,34 @@ def periodogram(x, fs=1.0, window=None, nfft=None, detrend='constant',
     x : array_like
         Time series of measurement values
     fs : float, optional
-        Sampling frequency of the `x` time series. Defaults to 1.0.
+        Sampling frequency of the ``x`` time series. Defaults to 1.0.
     window : str or tuple or array_like, optional
-        Desired window to use. See `get_window` for a list of windows and
-        required parameters. If `window` is an array it will be used
-        directly as the window. Defaults to None; equivalent to 'boxcar'.
+        Desired window to use. See `get_window` for a list of windows
+        and required parameters. If ``window`` is array_like it will be
+        used directly as the window and its length must be nperseg.
+        Defaults to ``None``; equivalent to 'boxcar'.
```

Comment:
We should probably just change this to `'boxcar'` as it's more explicit (but still allow None->'boxcar' alias for backward compatibility)